app/models/central_contact.rb

```diff
@@ -7,12 +7,12 @@ def self.create_all_from(opts)
 
   def self.contacts(opts)
     opts[:xml].xpath('//overall_contact').collect{|xml|
-      CentralContact.new.create_from({xml: xml, :nct_id=>opts[:nct_id], contact_type: 'primary'})}
+      CentralContact.new.create_from({:xml=>xml, :nct_id=>opts[:nct_id], :contact_type=>'primary'})}
```

Comment:
Any specific reason for this change? Using the hashrocket syntax isn't require anymore.